### ISSUE TITLE
Grub EFI on non-EFI server

### ISSUE THREAD

**Issue by mxdpeep** [webmin/webmin#2014]
Webmin allows to install EFI on non-EFI server, APT keeps it back.

The following packages have been kept back:
  grub-efi-amd64-bin

**Comment by iliajie** [comment]
You should not use Grub module on contemporary systems (with Grub 2). Webmin Grub module is deprecated.

**Comment by mxdpeep** [comment]
![Screenshot_20230930-091603](https://github.com/webmin/webmin/assets/75358/873fb653-25d8-4e5b-9f20-af2af4678a4c)

**Comment by iliajie** [comment]
Yes, this is correct. Webmin indeed allows you to install kept-back packages. You don't have to install it though.

**Comment by mxdpeep** [comment]
but when it happens - like by a mistake, you can break your server consistency :/

**Comment by iliajie** [comment]
We could, I think, indicate that the given package is kept-back. However, all kept-back packages (most of the time) are safe to install.

You should figure out why do you have that package offered by package manager at the first place.
